indexer/services/ender/src/validators/stateful-order-validator.ts

```
@@ -27,19 +30,14 @@ import { Validator } from './validator';
-      this.event.orderRemoval === undefined &&
-      this.event.conditionalOrderPlacement === undefined &&
-      this.event.conditionalOrderTriggered === undefined &&
-      this.event.longTermOrderPlacement === undefined
-    ) {
+    if (this.allOrderPlacementTypesUndefined()) {
       return this.logAndThrowParseMessageError(
         'One of orderPlace, orderRemoval, conditionalOrderPlacement, conditionalOrderTriggered, ' +
-        'longTermOrderPlacement must be defined in StatefulOrderEvent',
+        'longTermOrderPlacement, or twapOrderPlacement must be defined in StatefulOrderEvent',
         { event: this.event },
       );
```

Biome (2.1.2)

[error] The function should not return a value because its return type is void.

The function is here:

'void' signals the absence of value. The returned value is likely to be ignored by the caller.

(lint/correctness/noVoidTypeReturn)